crates/bevy_render/src/camera/camera_driver_node.rs

```diff
@@ -53,6 +53,8 @@ impl Node for CameraDriverNode {
             }
         }
 
+        let clear_color_global = world.get_resource::<ClearColor>().unwrap();
```

Comment:
If you're going to use get_resource() and unwrap(), might as well just use resource()